test/al_aws_collector_test.js

```diff
@@ -803,7 +803,7 @@ describe('al_aws_collector tests', function() {
 
         it('send logmsgs got failed', function(done) {
                 ingestCLogmsgsStub.restore();
-                let logmsgErr = {"errorType":"StatusCodeError","errorMessage":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","name":"StatusCodeError","statusCode":400,"message":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","error":"{\"error\":\"body encoding invalid\"}","options":{"method":"POST","url":"https://api.global-services.us-west-2.global.alertlogic.com/ingest/v1/48649/data/logmsgs"}};
+                let logmsgErr = {"errorType":"StatusCodeError","errorMessage":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","name":"StatusCodeError","statusCode":400,"message":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","error":"{\"error\":\"body encoding invalid\"}","response":{"status": 400 },"options":{"method":"POST","url":"https://api.global-services.us-west-2.global.alertlogic.com/ingest/v1/48649/data/logmsgs"}};
```

Comment:
just for completeness, should we remove `logmsgErr.statusCode` from the log error now that it's replaced with `logmsgErr.response.status`?

Reply:
removed logmsgErr.statusCode for completeness